packaging/datadog-agent/source/setup_agent.sh

```diff
@@ -389,18 +413,22 @@ tar -xz -C "$DD_HOME/agent" --strip-components 1 -f "$DD_HOME/agent.tar.gz"
 rm -f "$DD_HOME/agent.tar.gz"
 print_done
 
+# get the version from the actual config file in the branch
+AGENT_VERSION=$(get_from_py_config AGENT_VERSION)
+INTEGRATIONS_VERSION=${INTEGRATIONS_VERSION:-$AGENT_VERSION}
```

Comment:
could you document `INTEGRATIONS_VERSION` in this comment section? https://github.com/DataDog/dd-agent/blob/89cb32acb09419df38dfa16339f9986d322696ee/packaging/datadog-agent/source/setup_agent.sh#L23-L45 (and document which default version is set when it's not specified :) )